hawkbit-rest/hawkbit-rest-docs/src/test/java/org/eclipse/hawkbit/rest/documentation/MgmtApiModelProperties.java

```diff
@@ -55,10 +55,9 @@ public final class MgmtApiModelProperties {
     public static final String ARTIFACT_HASHES_SHA1 = "SHA1 hash of the artifact.";
     public static final String ARTIFACT_HASHES_MD5 = "MD5 hash of the artifact.";
     public static final String ARTIFACT_HASHES_SHA256 = "SHA256 hash of the artifact.";
-
-    public static final String ARTIFACT_DOWNLOAD_LINK = "Download link of the artifact.";
-
+    public static final String ARTIFACT_DOWNLOAD_LINK = "Download link of the artifact based on the chosen download url type parameter.";
     public static final String ARTIFACT_LIST = "List of artifacts of given software module.";
+    public static final String ARTIFACT_DOWNLOAD_URL_TYPE = "Type of the artifact download url. (can be default or cdn)";
```

Comment:
"default" is not a good name for the value representing the default. Maybe we can call it "direct" or "mgmt"? Then we can specify it as:
public static final String ARTIFACT_DOWNLOAD_URL_TYPE = "Type of the artifact download url (can be \"direct\" (default) or \"cdn\")";